src/ubuntu/14.04/crossdeps/Dockerfile

```diff
@@ -29,3 +29,14 @@ RUN apt-get update \
         liblldb-3.9-dev \
         lldb-3.9 \
     && rm -rf /var/lib/apt/lists/*
+
+# Install x86_arm crossgen dependencies
+RUN apt-get update \
```

Comment:
Do we need to run `apt-get update` twice?  Once here, and then once again after `dpkg --add-architecture i386`?

Reply:
Yes, it needs to be set up before and after because it was cleaned on the last run command.

Reply:
https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/38/files#diff-af12928698be1a28269893dc27404212R31 Will clean the apt cache. We will need to run apt-get update on each RUN command. After dpkg --add-architecture is called apt-get update is required to complete the command. See https://wiki.debian.org/Multiarch/HOWTO

Reply:
I understand that part.  Why is it necessary to call `apt-get update` before `dpkg --add-architecture i386`?  I'm not seeing how this would be required.  Why isn't the following sufficient?

```
RUN dpkg --add-architecture i386 \
    && apt-get update \
    && apt-get install -y \
...
```